### ISSUE TITLE
add esmodule export

### ISSUE THREAD

**Issue by vhoyer** [braintree/sanitize-url#51]

**Comment by hollabaq86** [comment]
Yeah it makes sense to esmodule support, but it would likely need to wait until we do a major version update of the library. Going to add this to our backlog, no ETA on when this work would get picked up.

for internal records, issue 2519

**Comment by vhoyer** [comment]
a major really? I feel like if could be a minor, albeit a big minor change, since we would only be adding the export, right? I don't think that adding the extra export would break the API in anyway.

**Comment by hemry11** [comment]
> a major really? I feel like if could be a minor, albeit a big minor change, since we would only be adding the export, right? I don't think that adding the extra export would break the API in anyway.

**Comment by hemry11** [comment]
hello evry  body i,m new here please i,m will come??

**Comment by sofuxro** [comment]
@tcurdt @hemry11 @vhoyer 
Create a fork and updated it to use `import`
https://www.npmjs.com/package/@sofuxro/sanitize-url
